### DISCUSSION TITLE
HA su VM esxi installato su un server remoto collegato in vpn alla rete di casa è possibile?

### DISCUSSION THREAD

**Discussion by maucarle** [indomus/forum#709] [HUB: Home Assistant]
Come da titolo,
 chiedo se qualcuno ha provato a installare HA su un server con VM esxi collegandolo con un client VPN alla LAN dove sono installati gli attuatori domotici da controllare, ovviamente sulla LAN ci sarà un VPN server per la connessione.
In questo modo posso lasciare un solo server acceso per tutte le installazioni di HA che controllano diversi appartamenti

grazie
Mauro

**Comment by sky7176** [comment]
Ciao,
io non ho l'installazione su una VM ma su raspberry 4 a casa, collegata in vpn con l'ufficio.
In ufficio ho dispositivi che vengono gestiti da Homeassistant installato a casa

Giuseppe

**Reply by maucarle** [reply]
Ciao, Giuseppe
Quello che hai fatto e' proprio quello che vorrei fare, che client VPN hai usato sul RB4? E' un addio-on?
grazie
mauro

**Reply by sky7176** [reply]
Ciao, scusa per il ritardo.
Io ho casa e ufficio collegati mediante una vpn net to net realizzata con due firewall
I firewall sono ipfire.
Giuseppe

**Comment by maucarle** [comment]
Ok chiaro
ottima soluzione
grazie

Il giorno lun 13 set 2021 alle ore 22:42 sky7176 ***@***.***>
ha scritto:
